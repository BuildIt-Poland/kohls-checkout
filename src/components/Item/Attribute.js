import styled from 'styled-components';

import { fontSmall } from '../../styles/designTokens';

const Attribute = styled.label`
  font-size: ${fontSmall};
  display: block;
`;

export default Attribute;
