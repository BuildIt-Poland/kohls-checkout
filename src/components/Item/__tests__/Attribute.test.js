import React from 'react';
import { create } from 'react-test-renderer';

import Attribute from '../Attribute';

describe('COMPONENT - Attribute', () => {
  it('renders Attribute component', () => {
    const component = create(<Attribute />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
