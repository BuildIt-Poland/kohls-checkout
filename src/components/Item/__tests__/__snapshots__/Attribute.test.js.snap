// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Attribute renders Attribute component 1`] = `
.c0 {
  font-size: 1.2rem;
  display: block;
}

<label
  className="c0"
/>
`;
